test/unit/logic/transactionPool.js

```diff
@@ -0,0 +1,119 @@
+var TransactionPool  = require('../../../logic/transactionPool.js');
+var TransactionLogic = require('../../../logic/transaction.js');
+var DelegateModule   = require('../../../modules/delegates.js');
+
+var TransferLogic = require('../../../logic/transfer.js');
+
+var transactionTypes = require('../../../helpers/transactionTypes');
+var AccountModule    = require('../../../modules/accounts.js');
+var BlocksModule     = require('../../../modules/blocks.js');
+var AccountLogic     = require('../../../logic/account.js');
+var modulesLoader    = require('../../common/initModule').modulesLoader;
+var async            = require('async');
+var expect           = require('chai').expect;
+var node             = require('../../node');
+
+
+
+describe('transactionPool', function () {
+	var txPool;
+
+	before(function (done) {
+		// Init transaction logic
+		async.auto({
+
+			accountLogic    : function (cb) {
+				modulesLoader.initLogicWithDb(AccountLogic, cb);
+			},
+			blockModule     : ['accountLogic', function (result, cb) {
+				modulesLoader.initModuleWithDb(BlocksModule, cb, {
+					logic: { /* dependencies not included */ },
+				});
+			}],
+			transactionLogic: ['accountLogic', function (result, cb) {
+				modulesLoader.initLogicWithDb(TransactionLogic, cb, {
+					account: result.accountLogic
+				});
+			}],
+			delegateModule  : ['transactionLogic', function (result, cb) {
+				modulesLoader.initModuleWithDb(DelegateModule, cb, {
+					logic: {
+						transaction: result.transactionLogic
+					}
+				});
+			}],
+		}, function (err, result) {
+			modulesLoader.initModuleWithDb(AccountModule, function (err, __accountModule) {
+				expect(err).to.not.exist;
+
+				var account  = __accountModule;
+				var accountLogic = result.accountLogic;
+
+				// for correctly initializing setting blocks module
+				result.blockModule.lastBlock.set({ height: 10 });
+
+				result.delegateModule.onBind({
+					accounts: __accountModule,
+					// transactions: result.transactionModule,
+					blocks  : result.blockModule
+				});
+				var sendLogic = result.transactionLogic.attachAssetType(transactionTypes.SEND, new TransferLogic());
+				sendLogic.bind(account, /* rounds */ null);
```

Comment:
I think it's best to leave it like that for clarity as proper initialization for further testing might need rounds too.